### DISCUSSION TITLE
Docker hub matching code?

### DISCUSSION THREAD

**Discussion by BrianLakstins** [prodrigestivill/docker-postgres-backup-local#118] [Q&A] [answered]
The docker images on hub.docker.com have a volume for /var/lib/postgresql/data

I can't find any reference in the code to creating a volume for /var/lib/postgresql/data

Are the scripts that create the images on hub.docker.com creating the images based on the code, or am I missing something about the build process?

**Comment by BrianLakstins** [comment]
Never mind.  That volume is from the postgres image you are basing the backup image on.  I wonder if there is a "remove volume" type of docker build command that can get rid of it, since it's not needed.

**Comment by prodrigestivill** [comment, marked as answer]
Duplicate of #53 & #70 .

The [source code](https://github.com/docker-library/postgres/blob/271cf940d0b8e212d16309271d49a8fdd4f48978/Dockerfile-debian.template#L184) you are looking is in the parent image (`FROM` statement) from docker hub official library.
